### ISSUE TITLE
Trip polyline fails WCAG contrast test

### ISSUE THREAD

**Issue by ualch9** [OneBusAway/OBAKit#357]
![110175741-1df4c600-7db7-11eb-842c-70d9a86f327e](https://user-images.githubusercontent.com/22162410/110702171-08144600-81a7-11eb-8d89-437423e0ce49.jpeg)

OneBusAway OBAKit `brand` color: `#9EB74F`
Contrast ratio with standard background `#F8F5EC`: 2.06:1, FAIL
Contrast ratio with nature `#E6F1D1`: 1.91:1, FAIL

WCAG AA (for user interface components) requires contrast of 3:1.

Although the Apple Maps components themselves don't pass, we should at least try to make our polyline slightly more distinguishable.

Potential fixes:
- Adjust OBAKit `brand` color to be darker
- Leave as-is, but provide a different color, add outline, or add shadow, if the user has "Increase Contrast" iOS setting on

**Comment by barbeau** [comment]
FWIW, OBA Android just uses the GTFS route color for the line on the map.

**Comment by barbeau** [comment]
Also, technically the official OBA brand color is `#78aa36`:
https://onebusaway.org/brand-guidelines/

...but neither iOS nor Android used that exact color in-app. On Android it was changed to be more "material".

**Comment by ualch9** [comment]
> Also, technically the official OBA brand color is `#78aa36`:
> 
> https://onebusaway.org/brand-guidelines/
> 
> 
> 
> ...but neither iOS nor Android used that exact color in-app. On Android it was changed to be more "material".

I meant the color used on iOS, which is internally named "brand". I updated the root comment to clarify this.
